app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :baskets
```

Comment:
Do baskets and subscriptions need to be dependent as well ?

Reply:
Agreed, will add dependency